db/src/main/kotlin/edu/artic/db/Accessiblable.kt

```diff
@@ -0,0 +1,8 @@
+package edu.artic.db
+
+/**
+ * Used to ensure content that requires content description can use this interface
+ */
+interface Accessiblable {
```

Comment:
Rename to `HasContentDescription`, maybe? `AccessibilityAware`? I'm not keen on `Accessiblable`.